standby/src/lib.rs

```diff
@@ -1039,28 +1075,131 @@ impl Standby {
 
                     bystander.sender.replace(Sender::Stream(tx));
 
-                    ProcessStatus::Incomplete
+                    ProcessStatus::SentStream
                 } else {
-                    ProcessStatus::Complete
+                    ProcessStatus::Dropped
                 }
             }
         }
     }
 }
+/// Number of [`Standby`] calls that were completed.
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct ProcessResults {
+    /// Number of bystanders that were dropped due to the receiving end
+    /// dropping.
+    dropped: usize,
+    /// Number of future bystanders that were open and were sent an event.
+    fulfilled: usize,
+    /// Number of stream bystanders that were open and were sent an event.
+    sent: usize,
+}
+
+impl ProcessResults {
+    /// Create a new set of zeroed out results.
+    const fn new() -> Self {
+        Self {
+            dropped: 0,
+            fulfilled: 0,
+            sent: 0,
+        }
+    }
+
+    /// Number of [`Standby`] calls where the receiver had already dropped their
+    /// end.
+    ///
+    /// This may happen when a caller calls into [`Standby`] but then times out
+    /// or otherwise cancels their request.
+    pub const fn dropped(&self) -> usize {
```

Comment:
If this becomes `Copy`, this should be changed to take `self` instead of `&self`